### ISSUE TITLE
Ability to disable Plausible/telemetry

### ISSUE THREAD

**Issue by knadh** [makeplane/plane#1694]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Hi. We spotted our local installation sending requests to an external Plausible instance. As we're in a heavily regulated environment, any external telemetry/analytics requests could be a compliance issue. While any telemetry ideally should ideally be opt-in, an option to disable it would be great. Something as simple as an environment variable will do.

Thank you.

### Why should this be worked on?

Telemetry / tracking is a compliance issue in regulated environments (and some jurisdictions even).

**Comment by vamsi** [comment]
Thanks @knadh for bringing this up to our Notice. We are adding Plausible to our ENV variable and disabling it on a priority basis.

**Comment by vamsi** [comment]
@knadh, we have addressed this issue in #1696. I want to assure you that we don't collect any data from self-hosted instances. After inspecting the relevant Plausible code, we realized that Plausible only sends us metrics based on the data domain attribute, which was set to app.plane.so. However, to prevent sending any events, we've incorporated an environment validation. Your effort in highlighting this critical issue is greatly appreciated!

We will be updating our Docker images with the revised code in the next few hours. Thanks again!
